docs/changelog.md

```diff
@@ -1,4 +1,9 @@
-# CHANGELOG
+---
```

Comment:
Similar to the `CONTRIBUTING.md` file comment, would it be possible to import `CHANGELOG.md` into `docs/changelog.md` instead of moving it?

Thoughts being, seeing a `CHANGELOG` file in the directory root is a relatively common convention.